javascript/index.js

```diff
@@ -14,52 +16,91 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  printSeconds();
+  printMinutes();
 }
 
 function printMinutes() {
-  // ... your code goes here
+  minUni.innerHTML = (chronometer.getMinutes())%10;
+  minDec.innerHTML = Math.floor((chronometer.getMinutes())/10);
 }
 
 function printSeconds() {
-  // ... your code goes here
+  secUni.innerHTML = (chronometer.getSeconds())%10;
+  secDec.innerHTML = Math.floor((chronometer.getSeconds())/10);
 }
 
 // ==> BONUS
 function printMilliseconds() {
-  // ... your code goes here
-}
-
-function printSplit() {
-  // ... your code goes here
+  milUni.innerHTML = (chronometer.getMiliSecons())%10;
+  milDec.innerHTML = Math.floor((chronometer.getMiliSecons())/10)
 }
+//Are these necesary?
```

Comment:
Not necessary but they work for having a more modular code. You could put the dom lines that add the clear everything or add the split time to the dom here and just call this functions in the event listeners.